docs/src/use/configure/rules.md

```diff
@@ -258,7 +284,9 @@ foo(); // eslint-disable-line example/rule-name
 foo(); /* eslint-disable-line example/rule-name */
 ```
 
-**Note:** Comments that disable warnings for a portion of a file tell ESLint not to report rule violations for the disabled code. ESLint still parses the entire file, however, so disabled code still needs to be syntactically valid JavaScript.
+::: note
```

Comment:
There's no `note`, we currently have only `warning`, `important`, or `tip`.

![image](https://github.com/eslint/eslint/assets/44349756/6239c4f8-7918-4257-8b97-565944a09860)

Reply:
Thanks